pkg/broker/ingress/handler.go

```diff
@@ -113,20 +113,24 @@ func (h *Handler) ServeHTTP(response nethttp.ResponseWriter, request *nethttp.Re
 	}
 
 	ctx := request.Context()
-	h.logger.Debug("Serving http", zap.Any("headers", request.Header))
+	ctx = logging.WithLogger(ctx, h.logger)
+	if span := trace.FromContext(ctx); span != nil {
+		ctx = tracing.WithLogging(ctx, span)
+	}
+	logging.FromContext(ctx).Debug("Serving http", zap.Any("headers", request.Header))
```

Comment:
Is this intended to stay in?

Reply:
Yes, this was already present, I've just changed it to use the logger from ctx.